### DISCUSSION TITLE
Subscription and multiple schemas

### DISCUSSION THREAD

**Discussion by MarnixSnake** [ChilliCream/hotchocolate#6563] [Q&A]
I've made a new setup and everything is working fine for me:
```
services
	.AddGraphQLServer()
	.AddQueryType<Query>()
	.AddMutationType<Mutation>()
	.AddSubscriptionType<Subscription>()
	.AddMutationConventions()
	.AddInMemorySubscriptions();
```

But this will be part of multiple schemas. I used _AddLocalSchema_ to combine multiple schema's. The new setup:

```
string schemaName = "NameOfTheSchema";

services
	.AddGraphQLServer(schemaName)
	.AddQueryType<Query>()
	.AddMutationType<Mutation>()
	.AddSubscriptionType<Subscription>()
	.AddMutationConventions()
	.AddInMemorySubscriptions();

services
	.AddGraphQLServer()
	.AddLocalSchema(schemaName);
```
 
Note that this is working fine with query and mutation, but that I'm receiving an error on the subscription: "You must declare a subscribe resolver for subscription fields." Any suggestions to declare the resolver?

**Comment by VasRoko** [comment]
I've had similar issue but resolved, are you able to share what's in your `Subscription` class?
